Contardo/template/buggy_function.py

```diff
@@ -9,6 +9,7 @@ def angle_to_sexigesimal(angle_in_degrees, decimals=3):
     ----------
     angle_in_degrees : float
         A scalar angle, expressed in degrees
+    decimal : 
```

Comment:
I don't see any changes in the code. Only the notes.
Try ti check the angles->RA conversion.